.github/workflows/Tests.yml

```diff
@@ -0,0 +1,33 @@
+name: Tests
+on: [push, pull_request]
+jobs:
+  test:
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        julia-version: [1.0, 1.5]
+        os: [ubuntu-latest]
+    steps:
+      - uses: actions/checkout@v1.0.0
+      - uses: azure/login@v1.1
+        with:
+          creds: ${{ secrets.AZURE_CREDENTIALS }}
+      - name: Run Az CLI script
+        run: |
+          az group create -l southcentralus -n "${{ matrix.os }}-${{ matrix.julia-version }}-${{ github.run_id }}"
+      - uses: julia-actions/setup-julia@latest
+        with:
+          version: ${{ matrix.julia-version }}
+      - uses: julia-actions/julia-buildpkg@latest #change lastest to master
+      - uses: julia-actions/julia-runtest@latest #change lastest to master
```

Comment:
before this step will want an ssh-keygen step, this action exists that ensure that the image has ssh-keygen installed: https://github.com/shimataro/ssh-key-action, which should make the step pretty easy